src/main/java/org/grouplens/grapht/util/LogContext.java

```diff
@@ -0,0 +1,46 @@
+package org.grouplens.grapht.util;
+/**
+ * Created by Imran Kazi on 2/9/2015.
+ */
```

Comment:
We don't need this comment, just the JavaDoc comment below on the `LogContext` class. We generally don't put individual author names in header files (Git commit logs are adequate to show authorship).